burrow/src/daemon/net/apple.rs

```diff
@@ -1,10 +1,13 @@
 use std::thread;
+
 use tokio::runtime::Runtime;
-use tracing::error;
+use tracing::{error, info};
+
 use crate::daemon::{daemon_main, DaemonClient};
 
 #[no_mangle]
-pub extern "C" fn start_srv(){
+pub extern "C" fn start_srv() {
+    info!("Rust: Starting server");
```

Comment:
I don't think we need "Rust" in there necessarily
```suggestion
    info!("Starting server");
```